src/sparsebase/preprocess/preprocess.cc

```diff
@@ -1168,7 +1174,8 @@ bool GrayReorder<IDType, NNZType, ValueType>::is_banded(int nnz, int n_cols, NNZ
   if (double(band_count) / nnz >= 0.3) {
     banded = true;
   }
-  //std::cout << "NNZ % in band: " << double(band_count) / nnz << std::endl;
+  utils::Logger logger(typeid(GrayReorder));
```

Comment:
When the logger is used multiple times in a class, you might want to make it a member variable.

Reply:
Not related to this PR just something to keep in mind for later